modules/37_industry/subsectors/equations.gms

```diff
@@ -142,4 +156,54 @@ q37_costCESmarkup(t,regi,in)$(ppfen_industry_dyn37(in))..
   * (vm_cesIO(t,regi,in) + pm_cesdata(t,regi,in,"offset_quantity"))
 ;
 
+* lower bound on feso/feli/fega in chemicals FE input for feedstocks
+q37_chemicals_feedstocks_limit(t,regi)$( t.val ge cm_startyear ) .. 
+  sum(in_chemicals_37(in), vm_cesIO(t,regi,in))
+  =g=
+    sum(ces_eff_target_dyn37("ue_chemicals",in), vm_cesIO(t,regi,in))
+  * p37_chemicals_feedstock_share(t,regi)
+;
+
+*Flow of non-energy feedstocks. It is used for emissions accounting 
+q37_demFeFeedstockChemIndst(ttot,regi,entyFe,emiMkt)$(    ttot.val ge cm_startyear 
```

Comment:
you have already the second equation here, so I don't know why would you need to add the non energy terms to the q37_demFeIndst equation.